From: Kurt Roeckx <kurt@roeckx.be>
Subject: Change the perl path's to /usr/bin/perl

This is the result of running:
perl util/perlpath.pl /usr/bin

The upstream sources have this set to various different paths.

--- openssl-0.9.8m.orig/Configure
+++ openssl-0.9.8m/Configure
@@ -1,4 +1,4 @@
-:
+#!/usr/bin/perl
 eval 'exec perl -S $0 ${1+"$@"}'
     if $running_under_some_shell;
 ##
--- openssl-0.9.8m.orig/VMS/VMSify-conf.pl
+++ openssl-0.9.8m/VMS/VMSify-conf.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/perl
+#!/usr/bin/perl
 
 use strict;
 use warnings;
--- openssl-0.9.8m.orig/Netware/do_tests.pl
+++ openssl-0.9.8m/Netware/do_tests.pl
@@ -1,4 +1,4 @@
-# perl script to run OpenSSL tests
+#!/usr/bin/perl
 
 
 my $base_path      = "\\openssl";
--- openssl-0.9.8m.orig/apps/progs.pl
+++ openssl-0.9.8m/apps/progs.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 print "/* apps/progs.h */\n";
 print "/* automatically generated by progs.pl for openssl.c */\n\n";
--- openssl-0.9.8m.orig/os2/backwardify.pl
+++ openssl-0.9.8m/os2/backwardify.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl
 use strict;
 
 # Use as $0
--- openssl-0.9.8m.orig/times/091/mips-rel.pl
+++ openssl-0.9.8m/times/091/mips-rel.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 &doit(100,"Pentium   100 32",0.0195,0.1000,0.6406,4.6100);	# pentium-100
 &doit(200,"PPro      200 32",0.0070,0.0340,0.2087,1.4700);	# pentium-100
--- openssl-0.9.8m.orig/fips/mkfipsscr.pl
+++ openssl-0.9.8m/fips/mkfipsscr.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 # Quick & dirty utility to generate a script for executing the
 # FIPS 140-2 CMVP algorithm tests based on the pathnames of
 # input algorithm test files actually present (the unqualified
--- openssl-0.9.8m.orig/fips/fipsalgtest.pl
+++ openssl-0.9.8m/fips/fipsalgtest.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl
 # Perl utility to run or verify FIPS 140-2 CMVP algorithm tests based on the
 # pathnames of input algorithm test files actually present (the unqualified
 # file names are consistent but the pathnames are not).
--- openssl-0.9.8m.orig/ms/uplink.pl
+++ openssl-0.9.8m/ms/uplink.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # For Microsoft CL this is implemented as inline assembler. So that
 # even though this script can generate even Win32 code, we'll be
--- openssl-0.9.8m.orig/ms/segrenam.pl
+++ openssl-0.9.8m/ms/segrenam.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 my $quiet = 1;
 
--- openssl-0.9.8m.orig/ms/cmp.pl
+++ openssl-0.9.8m/ms/cmp.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 ($#ARGV == 1) || die "usage: cmp.pl <file1> <file2>\n";
 
--- openssl-0.9.8m.orig/test/cms-test.pl
+++ openssl-0.9.8m/test/cms-test.pl
@@ -1,4 +1,4 @@
-# test/cms-test.pl
+#!/usr/bin/perl
 # Written by Dr Stephen N Henson (steve@openssl.org) for the OpenSSL
 # project.
 #
--- openssl-0.9.8m.orig/test/cms-examples.pl
+++ openssl-0.9.8m/test/cms-examples.pl
@@ -1,4 +1,4 @@
-# test/cms-examples.pl
+#!/usr/bin/perl
 # Written by Dr Stephen N Henson (steve@openssl.org) for the OpenSSL
 # project.
 #
--- openssl-0.9.8m.orig/demos/b64.pl
+++ openssl-0.9.8m/demos/b64.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 #
 # Make PEM encoded data have lines of 64 bytes of data
--- openssl-0.9.8m.orig/demos/tunala/configure.in
+++ openssl-0.9.8m/demos/tunala/configure.in
@@ -1,4 +1,4 @@
-dnl Process this file with autoconf to produce a configure script.
+#!/usr/bin/perl
 AC_INIT(tunala.c)
 AM_CONFIG_HEADER(config.h)
 AM_INIT_AUTOMAKE(tunala, 0.0.1-dev)
--- openssl-0.9.8m.orig/crypto/x86cpuid.pl
+++ openssl-0.9.8m/crypto/x86cpuid.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/x86_64cpuid.pl
+++ openssl-0.9.8m/crypto/x86_64cpuid.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 $output=shift;
 $masm=1 if ($output =~ /\.asm/);
--- openssl-0.9.8m.orig/crypto/md5/asm/md5-586.pl
+++ openssl-0.9.8m/crypto/md5/asm/md5-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # Normal is the
 # md5_block_x86(MD5_CTX *c, ULONG *X);
--- openssl-0.9.8m.orig/crypto/md5/asm/md5-x86_64.pl
+++ openssl-0.9.8m/crypto/md5/asm/md5-x86_64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl
 #
 # MD5 optimized for AMD64.
 #
--- openssl-0.9.8m.orig/crypto/sha/asm/sha1-ia64.pl
+++ openssl-0.9.8m/crypto/sha/asm/sha1-ia64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/sha/asm/sha1-x86_64.pl
+++ openssl-0.9.8m/crypto/sha/asm/sha1-x86_64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/sha/asm/sha512-sse2.pl
+++ openssl-0.9.8m/crypto/sha/asm/sha512-sse2.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/sha/asm/sha512-ia64.pl
+++ openssl-0.9.8m/crypto/sha/asm/sha512-ia64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/sha/asm/sha512-x86_64.pl
+++ openssl-0.9.8m/crypto/sha/asm/sha512-x86_64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/sha/asm/sha1-586.pl
+++ openssl-0.9.8m/crypto/sha/asm/sha1-586.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # ====================================================================
 # [Re]written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/des/asm/des-586.pl
+++ openssl-0.9.8m/crypto/des/asm/des-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # The inner loop instruction sequence and the IP/FP modifications are from
 # Svend Olaf Mikkelsen <svolaf@inet.uni-c.dk>
--- openssl-0.9.8m.orig/crypto/des/asm/desboth.pl
+++ openssl-0.9.8m/crypto/des/asm/desboth.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 $L="edi";
 $R="esi";
--- openssl-0.9.8m.orig/crypto/des/asm/des686.pl
+++ openssl-0.9.8m/crypto/des/asm/des686.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 $prog="des686.pl";
 
--- openssl-0.9.8m.orig/crypto/des/asm/crypt586.pl
+++ openssl-0.9.8m/crypto/des/asm/crypt586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # The inner loop instruction sequence and the IP/FP modifications are from
 # Svend Olaf Mikkelsen <svolaf@inet.uni-c.dk>
--- openssl-0.9.8m.orig/crypto/lhash/num.pl
+++ openssl-0.9.8m/crypto/lhash/num.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 #node     10 ->   4
 
--- openssl-0.9.8m.orig/crypto/ripemd/asm/rmd-586.pl
+++ openssl-0.9.8m/crypto/ripemd/asm/rmd-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # Normal is the
 # ripemd160_block_asm_data_order(RIPEMD160_CTX *c, ULONG *X,int blocks);
--- openssl-0.9.8m.orig/crypto/rc4/asm/rc4-586.pl
+++ openssl-0.9.8m/crypto/rc4/asm/rc4-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # At some point it became apparent that the original SSLeay RC4
 # assembler implementation performs suboptimaly on latest IA-32
--- openssl-0.9.8m.orig/crypto/rc4/asm/rc4-x86_64.pl
+++ openssl-0.9.8m/crypto/rc4/asm/rc4-x86_64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/cast/asm/cast-586.pl
+++ openssl-0.9.8m/crypto/cast/asm/cast-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # define for pentium pro friendly version
 $ppro=1;
--- openssl-0.9.8m.orig/crypto/rc5/asm/rc5-586.pl
+++ openssl-0.9.8m/crypto/rc5/asm/rc5-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm","../../perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/perlasm/x86ms.pl
+++ openssl-0.9.8m/crypto/perlasm/x86ms.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 package x86ms;
 
--- openssl-0.9.8m.orig/crypto/perlasm/x86asm.pl
+++ openssl-0.9.8m/crypto/perlasm/x86asm.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # require 'x86asm.pl';
 # &asm_init("cpp","des-586.pl");
--- openssl-0.9.8m.orig/crypto/perlasm/x86nasm.pl
+++ openssl-0.9.8m/crypto/perlasm/x86nasm.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 package x86nasm;
 
--- openssl-0.9.8m.orig/crypto/perlasm/x86unix.pl
+++ openssl-0.9.8m/crypto/perlasm/x86unix.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 package x86unix;	# GAS actually...
 
--- openssl-0.9.8m.orig/crypto/perlasm/cbc.pl
+++ openssl-0.9.8m/crypto/perlasm/cbc.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # void des_ncbc_encrypt(input, output, length, schedule, ivec, enc)
 # des_cblock (*input);
--- openssl-0.9.8m.orig/crypto/perlasm/x86_64-xlate.pl
+++ openssl-0.9.8m/crypto/perlasm/x86_64-xlate.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # Ascetic x86_64 AT&T to MASM assembler translator by <appro>.
 #
--- openssl-0.9.8m.orig/crypto/bf/asm/bf-686.pl
+++ openssl-0.9.8m/crypto/bf/asm/bf-686.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm","../../perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/bf/asm/bf-586.pl
+++ openssl-0.9.8m/crypto/bf/asm/bf-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm","../../perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/objects/objects.pl
+++ openssl-0.9.8m/crypto/objects/objects.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 open (NUMIN,"$ARGV[1]") || die "Can't open number file $ARGV[1]";
 $max_nid=0;
--- openssl-0.9.8m.orig/crypto/objects/obj_dat.pl
+++ openssl-0.9.8m/crypto/objects/obj_dat.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # fixes bug in floating point emulation on sparc64 when
 # this script produces off-by-one output on sparc64
--- openssl-0.9.8m.orig/crypto/conf/keysets.pl
+++ openssl-0.9.8m/crypto/conf/keysets.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 $NUMBER=0x01;
 $UPPER=0x02;
--- openssl-0.9.8m.orig/crypto/bn/bn_prime.pl
+++ openssl-0.9.8m/crypto/bn/bn_prime.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # bn_prime.pl
 
 $num=2048;
--- openssl-0.9.8m.orig/crypto/bn/asm/x86.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm","../../perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/bn/asm/ppc.pl
+++ openssl-0.9.8m/crypto/bn/asm/ppc.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # Implemented as a Perl wrapper as we want to support several different
 # architectures with single file. We pick up the target based on the
--- openssl-0.9.8m.orig/crypto/bn/asm/co-586.pl
+++ openssl-0.9.8m/crypto/bn/asm/co-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm","../../perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/bn/asm/bn-586.pl
+++ openssl-0.9.8m/crypto/bn/asm/bn-586.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 push(@INC,"perlasm","../../perlasm");
 require "x86asm.pl";
--- openssl-0.9.8m.orig/crypto/bn/asm/mo-586.pl
+++ openssl-0.9.8m/crypto/bn/asm/mo-586.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # This is crypto/bn/asm/x86-mont.pl (with asciz from crypto/perlasm/x86asm.pl)
 # from OpenSSL 0.9.9-dev 
--- openssl-0.9.8m.orig/crypto/bn/asm/x86_64-mont.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86_64-mont.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/comba.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/comba.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub mul_add_c
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/add.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/add.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub bn_add_words
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/mul.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/mul.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub bn_mul_words
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/mul_add.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/mul_add.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub bn_mul_add_words
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/sqr.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/sqr.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub bn_sqr_words
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/sub.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/sub.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub bn_sub_words
--- openssl-0.9.8m.orig/crypto/bn/asm/x86/div.pl
+++ openssl-0.9.8m/crypto/bn/asm/x86/div.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # x86 assember
 
 sub bn_div_words
--- openssl-0.9.8m.orig/crypto/aes/asm/aes-586.pl
+++ openssl-0.9.8m/crypto/aes/asm/aes-586.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/aes/asm/aes-x86_64.pl
+++ openssl-0.9.8m/crypto/aes/asm/aes-x86_64.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # ====================================================================
 # Written by Andy Polyakov <appro@fy.chalmers.se> for the OpenSSL
--- openssl-0.9.8m.orig/crypto/asn1/charmap.pl
+++ openssl-0.9.8m/crypto/asn1/charmap.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 
 use strict;
 
--- openssl-0.9.8m.orig/util/mksdef.pl
+++ openssl-0.9.8m/util/mksdef.pl
@@ -1,4 +1,4 @@
-
+#!/usr/bin/perl
 # Perl script to split libeay32.def into two distinct DEF files for use in
 # fipdso mode. It works out symbols in each case by running "link" command and
 # parsing the output to find the list of missing symbols then splitting
--- openssl-0.9.8m.orig/util/dirname.pl
+++ openssl-0.9.8m/util/dirname.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 if ($#ARGV < 0) {
     die "dirname.pl: too few arguments\n";
--- openssl-0.9.8m.orig/util/tab_num.pl
+++ openssl-0.9.8m/util/tab_num.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 $num=1;
 $width=40;
--- openssl-0.9.8m.orig/util/sp-diff.pl
+++ openssl-0.9.8m/util/sp-diff.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # This file takes as input, the files that have been output from
 # ssleay speed.
--- openssl-0.9.8m.orig/util/mkerr.pl
+++ openssl-0.9.8m/util/mkerr.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 
 my $config = "crypto/err/openssl.ec";
 my $debug = 0;
--- openssl-0.9.8m.orig/util/clean-depend.pl
+++ openssl-0.9.8m/util/clean-depend.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 # Clean the dependency list in a makefile of standard includes...
 # Written by Ben Laurie <ben@algroup.co.uk> 19 Jan 1999
 
--- openssl-0.9.8m.orig/util/add_cr.pl
+++ openssl-0.9.8m/util/add_cr.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # This adds a copyright message to a souce code file.
 # It also gets the file name correct.
--- openssl-0.9.8m.orig/util/pod2man.pl
+++ openssl-0.9.8m/util/pod2man.pl
@@ -1,4 +1,4 @@
-: #!/usr/bin/perl-5.005
+#!/usr/bin/perl
     eval 'exec /usr/bin/perl -S $0 ${1+"$@"}'
 	if $running_under_some_shell;
 
--- openssl-0.9.8m.orig/util/mkstack.pl
+++ openssl-0.9.8m/util/mkstack.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 
 # This is a utility that searches out "DECLARE_STACK_OF()"
 # declarations in .h and .c files, and updates/creates/replaces
--- openssl-0.9.8m.orig/util/selftest.pl
+++ openssl-0.9.8m/util/selftest.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 #
 # Run the test suite and generate a report
 #
--- openssl-0.9.8m.orig/util/ck_errf.pl
+++ openssl-0.9.8m/util/ck_errf.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # This is just a quick script to scan for cases where the 'error'
 # function name in a XXXerr() macro is wrong.
--- openssl-0.9.8m.orig/util/mklink.pl
+++ openssl-0.9.8m/util/mklink.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # mklink.pl
 
--- openssl-0.9.8m.orig/util/src-dep.pl
+++ openssl-0.9.8m/util/src-dep.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # we make up an array of
 # $file{function_name}=filename;
--- openssl-0.9.8m.orig/util/deleof.pl
+++ openssl-0.9.8m/util/deleof.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 while (<>)
 	{
--- openssl-0.9.8m.orig/util/arx.pl
+++ openssl-0.9.8m/util/arx.pl
@@ -1,4 +1,4 @@
-#!/bin/perl
+#!/usr/bin/perl
 
 # Simple perl script to wrap round "ar" program and exclude any
 # object files in the environment variable EXCL_OBJ
--- openssl-0.9.8m.orig/util/copy.pl
+++ openssl-0.9.8m/util/copy.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Fcntl;
 
--- openssl-0.9.8m.orig/util/mkdir-p.pl
+++ openssl-0.9.8m/util/mkdir-p.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # mkdir-p.pl
 
--- openssl-0.9.8m.orig/util/mkdef.pl
+++ openssl-0.9.8m/util/mkdef.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 #
 # generate a .def file
 #
--- openssl-0.9.8m.orig/util/files.pl
+++ openssl-0.9.8m/util/files.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # used to generate the file MINFO for use by util/mk1mf.pl
 # It is basically a list of all variables from the passed makefile
--- openssl-0.9.8m.orig/util/mkfiles.pl
+++ openssl-0.9.8m/util/mkfiles.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # This is a hacked version of files.pl for systems that can't do a 'make files'.
 # Do a perl util/mkminfo.pl >MINFO to build MINFO
--- openssl-0.9.8m.orig/util/perlpath.pl
+++ openssl-0.9.8m/util/perlpath.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # modify the '#!/usr/local/bin/perl'
 # line in all scripts that rely on perl.
--- openssl-0.9.8m.orig/util/mk1mf.pl
+++ openssl-0.9.8m/util/mk1mf.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # A bit of an evil hack but it post processes the file ../MINFO which
 # is generated by `make files` in the top directory.
 # This script outputs one mega makefile that has no shell stuff or any
--- openssl-0.9.8m.orig/util/err-ins.pl
+++ openssl-0.9.8m/util/err-ins.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # tack error codes onto the end of a file
 #
--- openssl-0.9.8m.orig/util/pl/Mingw32.pl
+++ openssl-0.9.8m/util/pl/Mingw32.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Mingw32.pl -- Mingw
 #
--- openssl-0.9.8m.orig/util/pl/unix.pl
+++ openssl-0.9.8m/util/pl/unix.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # unix.pl - the standard unix makefile stuff.
 #
--- openssl-0.9.8m.orig/util/pl/netware.pl
+++ openssl-0.9.8m/util/pl/netware.pl
@@ -1,4 +1,4 @@
-# Metrowerks Codewarrior or gcc / nlmconv for NetWare
+#!/usr/bin/perl
 #
 
 $version_header = "crypto/opensslv.h";
--- openssl-0.9.8m.orig/util/pl/VC-32.pl
+++ openssl-0.9.8m/util/pl/VC-32.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # VC-32.pl - unified script for Microsoft Visual C++, covering Win32,
 # Win64 and WinCE [follow $FLAVOR variable to trace the differences].
 #
--- openssl-0.9.8m.orig/util/pl/OS2-EMX.pl
+++ openssl-0.9.8m/util/pl/OS2-EMX.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # OS2-EMX.pl - for EMX GCC on OS/2
 #
--- openssl-0.9.8m.orig/util/pl/ultrix.pl
+++ openssl-0.9.8m/util/pl/ultrix.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # linux.pl - the standard unix makefile stuff.
 #
--- openssl-0.9.8m.orig/util/pl/linux.pl
+++ openssl-0.9.8m/util/pl/linux.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # linux.pl - the standard unix makefile stuff.
 #
--- openssl-0.9.8m.orig/util/pl/BC-32.pl
+++ openssl-0.9.8m/util/pl/BC-32.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # Borland C++ builder 3 and 4 -- Janez Jere <jj@void.si>
 #
 
